src/docs/JobAnalytics/JobAnalyticsUserGuide.adoc

```diff
@@ -0,0 +1,96 @@
+
+In the top side of the search window, a reminder of the selected filter values is displayed to show the filters that were considered during the latest search query.
+
+[[_JA-search-png]]
+image::JA-search.png[align=center]
+
+=== Execution Metrics
+As shown in the screenshot below, Job Analytics provides a summary of the most important job execution metrics. For instance, the dashboard shows:
+
+- A first panel to display the number of total jobs that correspond to the user query and filters. It also shows the ratio of successful jobs over the total number, and the number of jobs that are in progress and not yet finished.
+- A second summary panel to display the number of jobs with issues. We distinguish two types of issues: jobs that are finished but have encountered issues during their execution, and interrupted jobs that did not finish their execution and were stopped due to several kinds of issues such as insufficient resources, manual interruption, etc. So, we distinguish 4 types of interrupted states: `In-Error`, `Failed`, `Canceled`, and `Killed`.
+- The last metric gives an overview of the average duration of selected jobs.
+
+Users can also define their own job metrics but this feature is not fully implemented and will be integrated in the next release.
+
+image::JA-metrics.png[align=center]
+
+=== Job Charts
+
+Job Analytics includes three types of charts:
+
+- *Job duration chart:* This chart shows job duration per workflow instance. The x-axis shows the job ID and the y-axis shows the job duration. Hovering over the lines will also display the same information as a tooltip (see screenshot below). Using the duration chart will eventually help the users to identify any abnormal performance behavior among several workflow executions.
+[[_JA-duration]]
+
+image::JA-duration.png[align=center]
+- *Job variables chart:* This chart is intended to show all variable values of selected workflows. It helps to verify the workflow input values across all jobs. The chart provides the ability to hide or show specific input variables in the legend as shown in the figure below.
+
+- *Job results chart:* This chart is intended to show all result values of selected workflows. It helps to verify the workflow results across all jobs. The chart provides also the ability to hide or show specific results as shown in the figure above.
+
+image::JA-chart.png[align=center]
+
+All charts provides some advanced features such as "maximize" and "enlarge" to better view the results, and "move" to customize the dashboard layout (see top left side of charts). All of them provides the hovering feature as previously described and two types of charts to display: line and bar charts. Switching from one to another can be activated through a toggle button located in the top right of the chart. Same for show/hide variables and results.
+
+
+=== Job Execution table
```

Comment:
While I understand that "table" can be in lowercase, I would put in uppercase just for the sake of aesthetics, since it's the only word in all the subsection titles which is in lowercase.